src/components/LandingPage/Donation/ImgCarousel.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import img from "./image.jpg";
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+import { Carousel } from "react-responsive-carousel";
+import "./Donation.css";
+
+// Stateful component
+class ImgCarousel extends Component {
+  state = {
+    // An array of images
+    images: []
+  };
+  render() {
+    return (
+      <Carousel autoPlay="true" transitionTime="350" infiniteLoop="true">
```

Comment:
http://react-responsive-carousel.js.org/storybook/index.html?selectedKind=Carousel&selectedStory=center%20mode%20with%20touch&full=0&down=1&left=1&panelRight=0&downPanel=kadirahq%2Fstorybook-addon-actions%2Factions-panel is an example of the same package "react-responsive-carousel", and happens to be very similar to ours. Take a look, I'm pretty sure that you can implement it exactly as it's meant to be.